client/src/components/Meetup/Meetup.js

```diff
@@ -1,8 +1,689 @@
+      }
+      setIsUpdating(false);
+      setUserMeetupStatus("pending");
+    } else {
+      setShowLogin(true);
+    }
+  };
+
+  const renderAttendees = () => {
+    // https://github.com/mui-org/material-ui/blob/master/packages/material-ui-lab/src/AvatarGroup/AvatarGroup.js
+    // TODO: convert to modal expandable list when too many attendees
+
+    let attendeeDisplay;
+    if (attendees.length > 0) {
+      attendeeDisplay = (
+        <Grid item container justify="flex-start">
+          {attendees.map((attendee) => (
+            <Grid
+              key={attendee.id}
+              item
+              container
+              direction="column"
+              xs={2}
+              alignItems="center"
+              className={classes.attendee}
+              spacing={1}
+            >
+              <Grid item>
+                <Avatar
+                  className={classes.avatar}
+                  src={attendee.profilePic}
+                  component={RouterLink}
+                  to={`/user/${attendee.id === user.id ? "me" : attendee.id}`}
+                />
+              </Grid>
+              <Grid item>
+                <Typography
+                  variant="body2"
+                  className={classes.profileName}
+                  component={RouterLink}
+                  to={`/user/${attendee.id === user.id ? "me" : attendee.id}`}
+                >
+                  {attendee.name}
+                </Typography>
+              </Grid>
+            </Grid>
+          ))}
+        </Grid>
+      );
+    }
+
+    return (
+      <Grid
+        item
+        container
+        direction="column"
+        spacing={3}
+        className={classes.attendeeList}
+      >
+        <Grid item>
+          <Typography variant="body2">
+            Attendees:{" "}
+            {attendees.length === 0 && "there are currently no attendees."}
+          </Typography>
+        </Grid>
+        {attendeeDisplay}
+      </Grid>
+    );
+  };
+
+  const renderPendingAttendees = () => {
+    // https://github.com/mui-org/material-ui/blob/master/packages/material-ui-lab/src/AvatarGroup/AvatarGroup.js
+    // TODO: convert to modal expandable list when too many attendees
+
+    let pendingDisplay;
+    if (pendingAttendees.length > 0) {
+      pendingDisplay = (
+        <Grid item container justify="flex-start">
+          {pendingAttendees.map((attendee) => (
+            <Grid
+              key={attendee.id}
+              item
+              container
+              direction="column"
+              xs={2}
+              alignItems="center"
+              className={classes.attendee}
+              spacing={1}
+            >
+              <Grid item>
+                <Avatar className={classes.avatar} src={attendee.profilePic} />
+              </Grid>
+              <Grid item>
+                <Typography variant="body2">{attendee.name}</Typography>
+              </Grid>
```

Comment:
Is there a reason we don't have links for the pending attendees? If we include them, seems like we can extract some shared stuff with the `attendeeDisplay` things into a separate function (up to you).